### PR TITLE
DAOS-9562 test: Support VMD clusters w/ non-Optane NVMes

### PR DESCRIPTION
**Pull request by phender**
Adding support for an updated default launch.py --nvme=auto:-3DNAND
argument to support inverted matching for newer clusters in CI.

Signed-off-by: Phillip Henderson <phillip.henderson@intel.com>

### CODE REVIEW

**Comment by phender** [comment]
This cannot land until the following have been merged:
- [DAOS_9562 test: Support VMD clusters w/ non-Optane NVMes (#8571) by phender · Pull Request #8651 · daos-stack/daos](https://github.com/daos-stack/daos/pull/8651) 
- [DAOS_9562 test: Support VMD clusters w/ non-Optane NVMes (#8571) by phender · Pull Request #8652 · daos-stack/daos](https://github.com/daos-stack/daos/pull/8652)

**Comment by phender** [comment]
All required daos changes have been landed.  This has change been tested in against the landed daos master branch changes successfully.

**Review by brianjmurrell — APPROVED** [review]

**Pull request merged**
